plugin

```diff
@@ -1 +1 @@
-Subproject commit ad539b56c5354c1083f479d4c8d275ae8c7fdfa7
```

Comment:
last plugin cocos2dx version is still 3.7, this commit is change version number to 3.9 in plugin.